x-pack/solutions/observability/plugins/observability/public/components/alerts_table/alerts_table.tsx

```diff
@@ -44,6 +44,8 @@ export function ObservabilityAlertsTable(props: ObservabilityAlertsTableProps) {
   const { observability } = useKibana<{ observability?: ObservabilityPublicStart }>().services;
   const { observabilityRuleTypeRegistry, config } = usePluginContext();
 
+  const AlertActions = getAlertActions({ extraRowActions: props.additionalContext?.extraRowActions });
```

Comment:
This `additionalContext` thing appears to be the only inlet we have for providing our own props to the alerts table, so I've used it. I will have definitely forgotten what it is or how it works by the time I'm back in a couple weeks, it's very confusing and indirect.